src/linear_search.ts

```diff
@@ -37,5 +37,10 @@ export default function linearSearch(
   haystack: number[],
   needle: number,
 ): boolean {
+  for(let i = 0; i < haystack.length; i++){
+    if(haystack[i] == needle){
```

Comment:
```suggestion
    if(haystack[i] === needle){
```
use triple equals just because js is crazy